### PR TITLE
Fix deprecated ginkgo flags

### PR DESCRIPTION
**Pull request by sohankunkerkar**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:

Fixes deprecated ginkgo flags

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

None

#### Does this PR introduce a user-facing change?

No

### CODE REVIEW

File: dependencies.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ dependencies:
       match: version
 
   - name: conmon
-    version: v2.1.0
+    version: v2.1.2
     refPaths:
     - path: scripts/versions
```
Comment (saschagrunert) [inline dependencies.yaml:39]: We have to change this file as well.

**Review by saschagrunert — APPROVED** [review]
LGTM
/ok-to-test

**Comment by sohankunkerkar** [comment]
Splitting this PR into two PRs to narrow down CI failures

**Comment by TomSweeneyRedHat** [comment]
LGTM
once you run down the test hiccups.

**Pull request closed without merging**